src/args.c

```diff
@@ -499,15 +499,26 @@ void parse_args(int argc, char* argv[])
 	}
 
 	// Local reverse name resolver
-	if(argc == 3 && strcasecmp(argv[1], "ptr") == 0)
+	if((argc == 3 || argc == 4) && strcasecmp(argv[1], "ptr") == 0)
```

Comment:
This new argument is not documented in the help text

![Screenshot at 2024-05-28 21-28-12](https://github.com/pi-hole/FTL/assets/26622301/456c9666-bc89-48da-ac63-37c0fcbe9cfd)

Reply:
![grafik](https://github.com/pi-hole/FTL/assets/16748619/ee238293-aaa6-4e57-9b02-9035f2cb31ef)